src/plone/initializer/setuphandlers.py

```diff
@@ -20,9 +20,10 @@ def getNonInstallableProfiles(self):
         ]
 
 
-def _load_senaite_data(context):
+def _load_setup_data(context):
+    context
     portal_setup = getToolByName(context, 'portal_setup')
-    tar_file_path =  os.path.join(os.path.dirname(getConfiguration().clienthome),'','importdata','data.tar.gz')
+    tar_file_path =  os.path.join(os.path.dirname(getConfiguration().clienthome),'importdata',getSite().id,'data.tar.gz')
```

Comment:
Should that be a change proposed to the main repo?

Reply:
This is a project we actually own it is not a fork.